conditions/condition_factory.py

```
@@ -288,21 +413,29 @@ def create_condition(cls, config: Dict[str, Any]) -> BaseCondition:
         Returns:
             생성된 조건 인스턴스
         """
+        # STDOUT 직접 출력 추가
+        print(f"[FACTORY] create_condition 호출됨! config={config}")
+
         condition_type = config.get("type")
+        print(f"[FACTORY] condition_type={condition_type}")
 
         if not condition_type:
             raise ValueError("조건 타입이 지정되지 않았습니다")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)